### PR TITLE
Add plugin repo 

### LINKED ISSUE #94: Add plugin repo back in

### PR DESCRIPTION
**Pull request by matmair**
Fixes #94 

This finishes the plugin repo, adds links to it in several places and updates the readme to the current state.

### CODE REVIEW

File: _repo/shopify.md [hunk 1]
```diff
@@ -1,12 +1,21 @@
...
+license: MIT                                                       # License, we prefer open source
+open_source: true                                                  # Is this project licensed with an OSI-approved license - aka 'open source'
+stable: true                                                       # Is this project stable? Should users deploy this in their instace?
+maintained: true                                                   # Is this project maintained?
+pypi: true                                                         # Is availanle via PyPi
+package_name: inventree-shopify                                    # Name of the package on the index, required if pypi true
+github: https://github.com/matmair/ShopifyIntegrationPlugin        # Ĺink to repo in GitHub, one of github, gitlab or source is required
+gitlab: https://gitlab.com/matmair/ShopifyIntegrationPlugin        # Ĺink to repo in Gitlab, one of github, gitlab or source is required
+source: https://git.mjmair.com/matmair/ShopifyIntegrationPlugin    # Link to source, one of github, gitlab or source is required
```
Comment (wolflu05) [inline _repo/shopify.md:12]: I am not sure if so much different source options are required here. (And btw. your own git server does not work.)

Comment (matmair) [reply _repo/shopify.md:12]: @wolflu05 the options are mainly there to allow us different install methods. GitHub currently does not offer to host python packages, [GitLab](https://docs.gitlab.com/ee/user/packages/pypi_repository/) and Gitea do. My private git infrastructure only works if you are in my SD-WAN, it is not public.

File: _repo/shopify.md [hunk 2]
```diff
@@ -1,12 +1,21 @@
 ---
-name: ShopifyIntegrationPlugin
-author: matmair
-website: https://mjmair.com
-github: https://github.com/matmair/ShopifyIntegrationPlugin
-categories:
+name: ShopifyIntegrationPlugin                                     # Name of the plugin (should be either NAME, TITLE or SLUG)
```
[the remark this answers is not in the document]
Comment (matmair) [reply _repo/shopify.md:2]: This file will be used as a template, I will move it soon. The comments are not needed for submission, I will sketch out the process in a blog entry.

File: _repo/shopify.md (hunk 1 above)
Comment (wolflu05) [reply _repo/shopify.md:12]: Yeah, but I am not sure if it's a good thing to provide a url that is not publicly available.

Comment (matmair) [reply _repo/shopify.md:12]: it is an example. And the system is pretty secure and monitored, I am not worried.

**Comment by matmair** [comment]
@SchrodingersGat this would be my MVP for the plugin repo. When creating the website I also added you as a publisher - could you review the stuff I put there?

**Comment by matmair** [comment]
@wolflu05 I have added a realistic example of how I imagine a listing looking here: https://github.com/inventree/inventree-website/pull/111

**Comment by SchrodingersGat** [comment]
@matmair looks nice, although is it possible to have the URL as "plugins" instead of "repo"? *repo* (in my mind) makes it sound like we are actually hosting all of these plugins

**Comment by matmair** [comment]
@SchrodingersGat i had a bit of a different idea: I changed the link to the pattern of /author/plugin_name - like github URLs. LMK what you think?

**Comment by SchrodingersGat** [comment]
image

**Comment by matmair** [comment]
@SchrodingersGat I added an example and fixed the rendering error

**Comment by SchrodingersGat** [comment]
Very nice :) I'd like to take a look at some visual improvements at some point, but really happy with what you have put together! This will be great for plugin visibility, excellent work @matmair

**Comment by matmair** [comment]
@SchrodingersGat happy for any suggestions, this is the best I can design - wise.

**Pull request merged**